whisper/stt/processing/streaming.py

```diff
@@ -100,14 +115,24 @@ def ws_streaming(websocket_server: WSServer, model_and_alignementmodel):
             logger.info(f"Connection closed by client: {e}")
             break
         if "eof" in str(message):
-            o = online.finish()
-            websocket_server.send(whisper_to_json(o))
             logger.info(f"End of stream {message}")
+            o, _ = online.process_iter()
+            logger.info(f"Last committed text: {o}")
+            b = online.finish()
+            logger.info(f"Last buffered text: {o}")
+            websocket_server.send(whisper_to_json([o, b]))
             websocket_server.close()
             break
-        online.insert_audio_chunk(bytes_to_array(message))
-        o, _ = online.process_iter()
-        websocket_server.send(whisper_to_json(o))
+        audio_chunk = bytes_to_array(message)
+        online.insert_audio_chunk(audio_chunk)
+        if online.get_buffer_size() >= STREAMING_MIN_CHUNK_SIZE:
```

Comment:
hmm, I already saw this piece of code.
Are you copy-pasting things around because there are two functions with different interfaces?
Isn't there a way to factor the code and share the common logic, for whatever interface?

Reply:
done, it seems to work